build.gradle

```diff
@@ -146,6 +153,24 @@ def convertGitBranch = { gitBranch ->
     }
 }
 
+createExe {
+    dontWrapJar = true
+    stayAlive = true
+    version = "${versionMajor}.${versionMinor}"
+    textVersion = displayVersion
+    companyName = orgName
+    jreMinVersion = "${targetCompatibility}.0" // launch4j requires it's in the form x.x.x, thus this currently renders to 1.8.0
```

Comment:
Uhm, I'm not sure how to fix this - JRE/JDK versions are in the form x.x.x_[xx] (where each x is a single digit), so if I try with `1.8.40` the build fails with `launch4j: Invalid data: Minimum JRE version should be x.x.x[_xx]`, what version should I exactly require? `1.8.0_40`? I'm sorry but I can't find any `build.sbt` file, or resources showing that a specific version is required for JavaFX.

Also if a specific update of Java 8 is required I'll have to change the JVM detection logic in the installer, because the `CurrentVersion` value in the registry key `HKLM\SOFTWARE\JavaSoft\Java Runtime Environment` only specifies `1.8`, while I have to look in the subkeys to find the update number (e.g. 121 on my machine which has JRE 1.8.0_121).